docs/variants/msi_z690/releases.md

```diff
@@ -14,6 +14,84 @@ For details about our release process please read
+- Memory speed is displayed in the firmware setup main page
+- [Support for taking screenshots in the firmware](https://docs.dasharo.com/dev-proc/screenshots/#taking-screenshots)
+- Microsoft and Windows 2023 UEFI Secure Boot certificates
+
+### Changed
+
+- [This is a Dasharo Entry Subscription release](https://docs.dasharo.com/dev-proc/versioning/#dasharo-entry-subscription-releases)
+- [Updated microcode to the newer version; refer to SBOM section below](https://github.com/coreboot/intel-microcode/commit/6788bb07eb5f9e9b83c31ea1364150fe898f450a)
+- [Updated ME to the newer version; refer to SBOM section below](https://github.com/Dasharo/dasharo-blobs/tree/main/msi/ms7d25)
+- [Switched to the Raptor Lake-S Client FSP; refer to SBOM section below](https://github.com/intel/FSP/tree/481ea7cf0bae0107c3e14aa746e52657647142f3/RaptorLakeFspBinPkg/Client/RaptorLakeS)
+- Get SMBIOS serial number and UUID from ROMHOLE region instead of CBFS
+  (ROMHOLE is preserved during FlashBIOS and updates via DTS)
+- Rebased coreboot on 4.21 tag
+- Reduced the amount of microcode blobs included in the build, due to
+  redundancy (some blobs had the same shasum and supported the same CPUIDs)
+- Enroll default UEFI Secure Boot keys on the first boot
+- [Improved UEFI Secure Boot menu user experience](https://docs.dasharo.com/dasharo-menu-docs/device-manager/#secure-boot-configuration)
+- Scope of reset to default hotkey to global in firmware setup
+- Booting DTS over iPXE via HTTPS
+
+### Fixed
+
+- [No audio playback via headset](https://github.com/Dasharo/dasharo-issues/issues/483)
+- [Auto Boot Time-out is reset to 0 when F9 is pressed](https://github.com/Dasharo/dasharo-issues/issues/513)
+- [Some RPL-S CPUs ( 0xB06F2 (RPL-S C0) or 0xB06F5 (RPL-S H0)) have a problem
+  with booting on v1.1.2](https://github.com/Dasharo/dasharo-issues/issues/496)
+- [Change boot order menu is confusing](https://github.com/Dasharo/dasharo-issues/issues/422)
+- [The setup menu does not issue a reset, resulting in saved but unapplied
+  settings](https://github.com/Dasharo/dasharo-issues/issues/398)
+- [PCI Express Resizable BAR programming](https://github.com/Dasharo/dasharo-issues/)
+- [PCI Express resource allocation for Intel ARC A750](https://github.com/Dasharo/dasharo-issues/issues/584)
+- P2SB BAR not properly reserved in ACPI
+- Power LED not blinking during S3 sleep
+- PS/2 controller not enabled at first boot after flashing
+- Incorrect C-states reported in ACPI unsupported by HW
+- Incorrect first timestamp format in ACPI FPDT
+
+### Known issues
+
+- [Windows 11 installer unable to detect i225 Ethernet NIC](https://github.com/Dasharo/dasharo-issues/issues/482)
+- [Cannot wake from suspend via RTC on QubesOS](https://github.com/Dasharo/dasharo-issues/issues/484)
+- [Windows 11 VBS (Virtualization-based Security) appears Not enabled on
+  System Information](https://github.com/Dasharo/dasharo-issues/issues/539)
+
```

Comment:
please add: 
- https://github.com/Dasharo/dasharo-issues/issues/484
- https://github.com/Dasharo/dasharo-issues/issues/521
- https://github.com/Dasharo/dasharo-issues/issues/483

Reply:
https://github.com/Dasharo/dasharo-issues/issues/483#issuecomment-1895454842

Reply:
Take a closer look at `Cannot wake from suspend via RTC on QubesOS` the issue 484 is already there.

Reply:
https://github.com/Dasharo/docs/pull/686/commits/f91807ba59f8f0d7c2b08ec1faba9a3311d888dc#diff-02c1965ab265070941dfe35aabf982d0946a2ee1545fac0763eda95ccf36162fR76